packages/playwright/types/test.d.ts

```diff
@@ -1838,6 +1850,8 @@ export type TestDetails = {
   annotation?: TestDetailsAnnotation | TestDetailsAnnotation[];
 }
 
+export type TestFilter = (tests: TestCase[]) => TestCase[];
```

Comment:
`(test: TestCase) => boolean` would probably be more straight forward here, but it also gives less context… e.g. number of test cases.

Maybe it's possible to support multiple signatures? e.g. `(test: TestCase) => boolean | (tests: TestCase[]) => TestCase[]` 
…but then it could be difficult to detect which one is used at runtime…

Reply:
Changed it in 5e212da2beb7bfe02af6f8077a1c039894ac1d8f to  ... https://github.com/microsoft/playwright/blob/5e212da2beb7bfe02af6f8077a1c039894ac1d8f/packages/playwright/types/test.d.ts#L1870-L1873